src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -38,6 +45,24 @@ class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface,
 
     public function __construct(array $defaultContext = [])
     {
+        $this->defaultContext = [
+            self::FORMAT_KEY => \DateTime::RFC3339,
+            self::TIMEZONE_KEY => null,
+        ];
+
+        if (!\is_array($defaultContext)) {
```

Comment:
`$defaultContext` is always an array as there is a type hint in the constructor signature.